resources/quality/z-bolt/bowden/ABS/v6 0.3/z-bolt_bowden_v6_0.30_ABS_extrafine.inst.cfg

```diff
@@ -0,0 +1,23 @@
+[general]
```

Comment:
A space in the folder name is not reliable. We do percent-encoding of these paths so that could lead to corruptions if using special characters like the space.

Reply:
Will be fixed soon, thanks.